packages/docs/SUMMARY.md

```diff
@@ -43,14 +44,14 @@
 
 ## Developer Guide
 
-- [Quick Start](developer-resources/start.md)
-  - [Hello Celo](developer-resources/walkthroughs/hellocelo.md)
-  - [Hello Contracts](developer-resources/walkthroughs/hellocontracts.md)
-  - [Hello Contract (Truffle + Remote Node)](developer-resources/walkthroughs/hello-contract-remote-node.md)
-  - [Hello Mobile Dapp](developer-resources/walkthroughs/hello-mobile-dapp.md)
+- [Overview](developer-resources/overview.md)
```

Comment:
you'll want to add redirects for the URLs that are going to change here

Reply:
I didn't change the path or the filename, just the title of the page. Will that still effect the URLs? Or should I change the file names to match the page titles and set up redirects? If so, I'd like some pointers about setting up redirects

Reply:
i found the redirects list in `gitbook.yaml` and added `celo-owner-guide/...: celo-holder-guide/...` which seems to work. The developer guide pages seem to work as well since I only changed page titles and didn't rename the files

Reply:
where is `gitbook.yaml`?